aspnetcore/fundamentals/startup.md

```diff
@@ -59,6 +59,34 @@ For features that require substantial setup, there are `Add[Service]` extension
 
 [!code-csharp[](../common/samples/WebApplication1/Startup.cs?highlight=4,7,11&start=40&end=55)]
 
+::: moniker range=">= aspnetcore-2.1"
+
+<a name="setcompatibilityversion"></a>
+### SetCompatibilityVersion
+
+The `SetCompatibilityVersion` method allows an app to opt-in or opt-out of breaking changes in ASP.NET Core APIs. By opting in, you get the latest behavior, and the long term behavior of ASP.NET Core.
+
+The following code sets the compatibility mode to ASP.NET Core 2.1:
+
+[!code-csharp[Main](startup/sampleCompatibility/Startup.cs?name=snippet1)]
+
+Applications that call `SetCompatibilityVersion(CompatibilityVersion.Version_2_0)` are protected from potentially breaking changes introduced in ASP.NET Core 2.1 and later 2.x versions . This protection does not extend to the next major version.
+
+The default compatibility for ASP.NET Core 2.1 and later  2.x applications that do **not** call `SetCompatibilityVersion` is 2.0 compatibility. That is, not calling `SetCompatibilityVersion` is the same as calling `SetCompatibilityVersion(CompatibilityVersion.Version_2_0)`.
+
+The following code sets the compatibility mode to ASP.NET Core 2.1, except for the following behavoirs:
+
+* [AllowCombiningAuthorizeFilters](https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNetCore.Mvc.Core/MvcOptions.cs)
+* [InputFormatterExceptionPolicy](https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNetCore.Mvc.Core/MvcOptions.cs)
+
+[!code-csharp[Main](startup/sampleCompatibility/Startup2.cs?name=snippet1)]
+
+The [MvcOptions](https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNetCore.Mvc.Core/MvcOptions.cs) class contains all the compatibility options you can set.
+
+Consider ASP.NET Core 3.0, which is not scheduled on the [ASP.NET Core Schedule and Roadmap](https://github.com/aspnet/Home/wiki/Roadmap). At some future date, there will be an ASP.NET Core 3.0 version. Applications targeting ASP.NET Core 3.0 will not be able to opt-out of breaking changes introduced in ASP.NET Core 2.x.
```

Comment:
I wouldn't say **not scheduled** - that makes it sound like there won't be a 3.0. I think this whole sentence could be cut out.

Reply:
You could add to here that the reason why we're comfortable removing things in 3.0 is that we think these are positive changes for **all** users, but might require changes to digest. Rather than springing a bunch of these changes on users in 3.0 the plan is to introduce them gently so users can benefit right away.